tox.ini

```diff
@@ -22,27 +22,27 @@ commands =
 [testenv:py27-functional]
 commands =
    python -V
-   {toxinidir}/scripts/kube-init.sh py.test -vvv -s []
+   {toxinidir}/scripts/kube-init.sh pytest -vvv -s []
 
 [testenv:py35-functional]
 commands =
    python -V
-   {toxinidir}/scripts/kube-init.sh py.test -vvv -s []
+   {toxinidir}/scripts/kube-init.sh pytest -vvv -s []
 
 [testenv:py36-functional]
 commands =
    python -V
-   {toxinidir}/scripts/kube-init.sh py.test -vvv -s []
+   {toxinidir}/scripts/kube-init.sh pytest -vvv -s []
 
 [testenv:py37-functional]
 commands =
    python -V
-   {toxinidir}/scripts/kube-init.sh py.test -vvv -s []
+   {toxinidir}/scripts/kube-init.sh pytest -vvv -s []
 
 [testenv:coverage]
 commands =
    python -V
-   nosetests --with-coverage --cover-package=kubernetes.config,kubernetes.watch --cover-tests
+   pytest --ignore=kubernetes/e2e_test --cov=kubernetes/watch --cov=kubernetes/config
```

Comment:
It looks like the tests for the dynamic client in base/dynamic are all e2e tests, which will be skipped by using pytest without a local cluster. If we are only testing the coverage for the watch and config folders we can skip those tests (for now).